staging/scripts/vault-prepare

```diff
@@ -1,9 +1,15 @@
 #!/usr/bin/env bash
+set -e
```

Comment:
This script is now made a little more robust with respect to where it is run from